src/components/sidebar/components/sidebar-nav-menu-item/index.tsx

```diff
@@ -116,7 +116,7 @@ const SidebarNavLinkItem = ({ item }: SidebarNavLinkItemProps) => {
 						<SidebarNavMenuItemBadge {...(item as $TSFixMe).badge} />
 					) : undefined
 				}
-				icon={isExternal ? <IconExternalLink16 /> : undefined}
```

Comment:
`SidebarNavMenuItem` now supports a `rightIcon` property. This seems necessary to render the progress icon.

Reply:
nit: it might be helpful for readability to pull out `icon` into a variable above the JSX given we've got two conditionals in its assignment now.

Reply:
sweet, thanks for these suggestions 💖 ✅ updated!